core/domain/exp_services.py

```diff
@@ -807,6 +807,9 @@ def _save_exploration(committer_id, exploration, commit_message, change_list):
 
     exploration.version += 1
 
+    # Update summary of changed exploration.
```

Comment:
A couple more thoughts on this: 

- Noticed that the other reason for this move was save_new_exploration (where this call used to live). Do you think I should just leave this call in twice, once in there and once in update_exploration? 
- I guess my larger question is why not have _save_exploration call an update on the exploration summary? Seems to make sense to me. Are you concerned about linking stuff up too much or something? 
- My final thought is that this seems like a bigger refactor than I'd like to focus on right now, since I'm still knee deep in implementing the impact calculation as well. Could I perhaps keep it as a nice-to-have to implement next week?